demisto_sdk/commands/format/tests/test_formatting_json_test.py

```diff
@@ -1216,17 +1235,20 @@ def test_set_fromversion_six_new_contributor_pack(self, pack):
         When
             - Run format command
         Then
-            - Ensure that the integration fromversion is set to 6.0.0
+            - Ensure that the integration fromversion is set to GENERAL_DEFAULT_FROMVERSION
         """
+        from demisto_sdk.commands.common.constants import \
+            GENERAL_DEFAULT_FROMVERSION
+
         pack.pack_metadata.update({'support': 'partner', 'currentVersion': '1.0.0'})
-        incident_type = pack.create_incident_type(name='TestType', content={'fromVersion': '5.5.0'})
-        incident_field = pack.create_incident_field(name='TestField', content={'fromVersion': '5.5.0'})
-        indicator_field = pack.create_indicator_field(name='TestFeild', content={'fromVersion': '5.5.0'})
-        indicator_type = pack.create_indicator_type(name='TestType', content={'fromVersion': '5.5.0'})
-        classifier = pack.create_classifier(name='TestClassifier', content={'fromVersion': '5.5.0'})
-        layout = pack.create_layout(name='TestLayout', content={'fromVersion': '5.5.0'})
```

Comment:
why did we remove the content arg?

Reply:
The purpose of the test is to check the correct assignment of the general default fromVersion key for new supported packs. Therefore, if we didn't remove the content arg the fromVersion will stay on 5.5.0 and the purpose of test will be compromised.